pkg/plugin/plugin.go

```diff
@@ -34,19 +34,33 @@ var (
 )
 
 // NewEdgeDBDatasource creates a new datasource instance.
-func NewEdgeDBDatasource(_ backend.DataSourceInstanceSettings) (instancemgmt.Instance, error) {
-	return &EdgeDBDatasource{}, nil
+func NewEdgeDBDatasource(dataSourceInstanceSettings backend.DataSourceInstanceSettings) (instancemgmt.Instance, error) {
+	opts := edgedb.Options{Concurrency: 4}
```

Comment:
Setting the concurrency is optional. If it is not set the value will be what is configured in EdgeDB. If EdgeDB doesn't have a value configured then it will be the number of CPUs or `4` whichever is larger. There is nothing wrong with passing a value here. I'm just over sharing :)

Reply:
It's good to know, thanks!